sycl/source/detail/program_manager/program_manager.hpp

```diff
@@ -498,6 +502,13 @@ class ProgramManager {
       std::map<std::vector<unsigned char>, ur_kernel_handle_t>;
   std::unordered_map<std::string, MaterializedEntries> m_MaterializedKernels;
 
+  // Holds bfloat16 device library images, the key is 0 for fallback version
+  // and 1 for native version. These bfloat16 device library images are
+  // provided by compiler long time ago, we expect no further update, so
+  // keeping 1 copy should be OK.
+  std::unordered_map<uint32_t, DynRTDeviceBinaryImageUPtr>
```

Comment:
Could this be just a `std::array`?

Reply:
Hi, @jopperm 
Yes, already updated the PR.
Thanks very much.